### DISCUSSION TITLE
Setting width for sections - V2

### DISCUSSION THREAD

**Discussion by anderer** [automadcms/discussions#23] [Q&A] [answered]
hello,

i am considering a switch from GRAV CMS to another one - and stumbled upon yours. In V1 I am able to set the width of sections - e.g. have 3 times 1/3 - but in V2 I am failing to set this. Where would I do it?

I know it is in Alpha - so just consider this as a UI/UX feedback : )

apart from that - i really like how clean everything is.

**Comment by anderer** [comment, marked as answer]
ok, sorry for the bother - i found it - and i am embarrased a little bit - it is now on the left - but still with the little dots in a square : )

**Reply by anderer** [reply]
ok, i at the end i dedicated almost this whole saturday to the automad - i like it more and more. looking forward to the official beta ov V2. When will you move from alpha to beta?

**Reply by marcantondahmen** [reply]
Thank you! No need to be embarrassed. Good feedback is always appreciated. The beta version is very close but there is not a fix date yet.

**Comment by Imerion** [comment]
This actually took me a while to find as well, but it feels logical now that I did. :)
It would be nice if the block settings (the small dots) were visible at all times though. As it is now I sometimes struggle with clicking on them since they disappear when the cursor is moving towards them due to accidentally hovering over another block. If they were visible at all times (perhaps a bit faded out if you want to avoid visual clutter) it would also help with discovery for this issue.
